tools/GraphClust/LocARNAGraphClust/mlocarna.xml

```diff
@@ -0,0 +1,189 @@
+<tool id="mlocarna" name="mlocarna" version="0.1.0" >
+	<requirements>
```

Comment:
@bgruening It's a wrapper that invokes locarna as one of the steps and pre/post-process input/output and creates custom files by reading tree, etc. So more or less needed. I would suggest @eteriSokhoyan to rename it something like: locarna_best_subtree ..